### ISSUE TITLE
Present design proposal for child theme

### ISSUE THREAD

**Issue by shawnr** [suwebdev/ncs-michigan#2]
Create and present design proposal for child theme to @reiners and @shawnr for review and approval. Proposal should include sample pages and design comps sufficient to convey design vision for major aspects of child theme.

**Comment by shawnr** [comment]
Deliver a PDF of design concepts showing comps of pages and elements. (IA referenced is: https://docs.google.com/document/d/1jX9KiAaHCjkdHV7_2Rt4e_6A26R6FxyNjBpYZLTdW08/edit)

* **Home Page** 
  * Include a section that allows for a breakout of persona presentation (reference https://ce.colorado.edu).
  * Pay attention to how we can present some kind of "global nav" or reference back to SU
  * Provide a place to put a video (general NCS advertising video)
  * Events list/feature
  * Suggest additional features/content/etc. as warranted

* **Header/Footer**
  * Provide mockup of links/menus/nav
  * Provide contact information in footer
  * Social media buttons (Facebook, LinkedIn, Twitter, Youtube)
  * Apply Now button

* **Top-tier from IA**
  * Section-specific landing pages
  * Each section probably needs a slightly different treatment

* **Program Brochures**
  * Each program will have three pages
  * Landing page will be big advertisement
    * Attractive landing
    * Brochure/guides downloads
    * Some section that draws people to the student success page
    * Apply now
  * Course page
    * List all courses in program with title, number, and description
  * Careers page
    * Career statistics, job titles, trivia, salary ranges
    * Basically work with the content that's on the career pages now

**Comment by jfehrenbacher** [comment]
Attached are two home page comps, one with a 'Programs' section and one without. 

I'm trying to bring some color into it with the slider, but if we're not into it that's an easy fix. I'm also using the 'Frame' font from the branding guidelines, but I'm not sure if we're even allowed to do that, as I've really never seen it anywhere else. I'm really fond, though. Essentially, the slider can absolutely change if it needs to, this is just what I have worked out right now.

Obviously all images are temporary. These are rough mockups, but any and all feedback is appreciated and extremely helpful. For the most part each section is based off of an element that I know can be found within the Michigan theme. Changing the hierarchy (for example, switching the 'Personas' section with the 'Programs' section is also easy and worth considering.

More coming tomorrow!

[HomeMockup.pdf](https://github.com/suwebdev/ncs-michigan/files/756277/HomeMockup.pdf)

[HomeWithPrograms.pdf](https://github.com/suwebdev/ncs-michigan/files/756278/HomeWithPrograms.pdf)

**Comment by reiners** [comment]
Hi Jack,
Thanks for sending these over. It’s really great to see things starting to come together.
As far as the slider, I really like the color. I think it looks nice. As far as the “Frame” font, while it’s the most interesting font we have in the SU branding guidelines, it’s used a lot in undergrad marketing, so I would like us to stay away from using it.
I would definitely like to switch the Personas and Programs sections, so that the Personas section is “above the fold” and the first thing someone sees to click on, as it hopefully will help inform their journey on our site. I am not sure we need a separate “Welcome” section, as I think we can add a paragraph above the personas that acts as the welcome. What do you think? Can we also swap programs and events so the events section is at the bottom?
Great start and looking forward to more! As always, let me know if you have any questions.
Thanks,
Sara

Sara Reiner | Director of Marketing and Communications
School of New and Continuing Studies | Seattle University
901 12th Avenue, Seattle, WA 98122-1090
Office: (206) 296-2461
seattleu.edu/ncs<http://www.seattleu.edu/ncs>
Get Social with NCS
 [cid:C4CC058A-0060-45D9-AE1B-06B8D899C107] <https://www.facebook.com/SeattleUNCS/> [cid:409621E6-56D6-47A6-8111-E608FE0CD9AC] <https://twitter.com/su_ncs> [cid:D122FC26-8383-4279-AE3F-129417BFBCB4] <https://www.linkedin.com/company-beta/15208682>

From: Jack Fehrenbacher
Reply-To: suwebdev/ncs-michigan
Date: Monday, February 6, 2017 at 5:01 PM
To: suwebdev/ncs-michigan
Cc: Sara Reiner, Mention
Subject: Re: [suwebdev/ncs-michigan] Present design proposal for child theme (#2)


Attached are two home page comps, one with a 'Programs' section and one without.

I'm trying to bring some color into it with the slider, but if we're not into it that's an easy fix. I'm also using the 'Frame' font from the branding guidelines, but I'm not sure if we're even allowed to do that, as I've really never seen it anywhere else. I'm really fond, though. Essentially, the slider can absolutely change if it needs to, this is just what I have worked out right now.

Obviously all images are temporary. These are rough mockups, but any and all feedback is appreciated and extremely helpful. For the most part each section is based off of an element that I know can be found within the Michigan theme. Changing the hierarchy (for example, switching the 'Personas' section with the 'Programs' section is also easy and worth considering.

More coming tomorrow!

HomeMockup.pdf<https://github.com/suwebdev/ncs-michigan/files/756277/HomeMockup.pdf>

HomeWithPrograms.pdf<https://github.com/suwebdev/ncs-michigan/files/756278/HomeWithPrograms.pdf>

**Comment by jfehrenbacher** [comment]
Yes, those are absolutely easy fixes! As for the 'Welcome' section, it's reasonable to do without it, particularly because there's room for welcome content in the slider or, as you mentioned, in the 'Personas' section. We would, however, need to find a new home for a video, if we're committed to having one on the home page. 

Another option would be to have a section that is only a video, or a 'Why NCS?' section that is structured like the current 'Welcome' section, and could direct to the 'About Us' page.

**Comment by reiners** [comment]
Hi Jack,
Where do you think a video would make the most sense from a design/user experience perspective on the landing page (if at all)?
Thanks,
Sara

________________________________

**Comment by jfehrenbacher** [comment]
I think that a video would make more sense on the 'About' landing page, assuming "Dean's Welcome" is getting its own page with its own video.  As we have the IA written out right now I'm a little confused as to what's supposed to go on the initial landing page for each section and what's supposed to have a page of its own (or if that's up to my own discretion).

Some clarification on that would go a long way.

**Comment by jfehrenbacher** [comment]
So here are two more versions of the home page, both with 'Personas' moved to the top, and with the 'Welcome' section removed. The 'Events' section has also been moved to the bottom. I made a second version of the 'Programs' section just in case you didn't like Personas & Programs looking so similar right next to one another. 

I've also attached an 'About NCS' landing page, home to the old 'Welcome' section as well as a simple 'Our Mission' section. I'm assuming we want the 'Dean's Welcome' to be its own page with its own video. Do we want any of the other 'About' content, i.e. 'Contact Us', 'Faculty & Staff', and 'Why Choose NCS?' to live on the landing page, or at least have sections pointing to them? What do you think?

[HomeNoWelcomeProgram1.pdf](https://github.com/suwebdev/ncs-michigan/files/762151/HomeNoWelcomeProgram1.pdf)
[HomeNoWelcomeProgram2.pdf](https://github.com/suwebdev/ncs-michigan/files/762150/HomeNoWelcomeProgram2.pdf)
[AboutNCS-1.pdf](https://github.com/suwebdev/ncs-michigan/files/762152/AboutNCS-1.pdf)

**Comment by jfehrenbacher** [comment]
I don't know what we're looking for as far as a landing page for the Information Center goes, so I was thinking it could be the 'Request Information' page that also has quicklinks to more specific informational pages. I just took from the current NCS page on the SU website as far as form content goes. Let me know if this is at all what you had in mind, or if this makes any sense.

[InfoCenter1.pdf](https://github.com/suwebdev/ncs-michigan/files/762336/InfoCenter1.pdf)

**Comment by reiners** [comment]
Hi Jack,
I really like this! Can we use this as the Contact Us form for the buttons that appear on the homepage?
Let me know what your think! :)
Thanks again,
Sara

Sara Reiner | Director of Marketing and Communications
School of New and Continuing Studies | Seattle University
901 12th Avenue, Seattle, WA 98122-1090
Office: (206) 296-2461
seattleu.edu/ncs<http://www.seattleu.edu/ncs>
Get Social with NCS
 [cid:F9FE3B2F-6350-47D3-9C6B-2ADCC961E31A] <https://www.facebook.com/SeattleUNCS/> [cid:8327C1E8-3CEA-4C01-8672-788BD2BED106] <https://twitter.com/su_ncs> [cid:4ABBE374-E3D6-4367-865F-E70CC3B042DF] <https://www.linkedin.com/company-beta/15208682>

From: Jack Fehrenbacher
Reply-To: suwebdev/ncs-michigan
Date: Wednesday, February 8, 2017 at 5:08 PM
To: suwebdev/ncs-michigan
Cc: Sara Reiner, Mention
Subject: Re: [suwebdev/ncs-michigan] Present design proposal for child theme (#2)


I don't know what we're looking for as far as a landing page for the Information Center goes, so I was thinking it could be the 'Request Information' page that also has quicklinks to more specific informational pages. I just took from the current NCS page on the SU website as far as form content goes. Let me know if this is at all what you had in mind, or if this makes any sense.

InfoCenter1.pdf<https://github.com/suwebdev/ncs-michigan/files/762336/InfoCenter1.pdf>

**Comment by jfehrenbacher** [comment]
As it stands right now the 'Contact' buttons in the theme default to a modal with a very simple form, but I'm sure we could either change the form in the child theme to reflect what's on the Request Information Page, or just have the 'Contact' button redirect to the Request Information page. 

Here are two versions of the 'Upcoming Events' page. The Michigan Theme uses 'The Events Calendar' plugin, which can be displayed as a list, grid, or calendar. I went with the list and calendar formats here, but we could look at the grid version if you prefer.

[UpcomingEventsCalendar.pdf](https://github.com/suwebdev/ncs-michigan/files/764793/UpcomingEventsCalendar.pdf)
[UpcomingEventsList.pdf](https://github.com/suwebdev/ncs-michigan/files/764792/UpcomingEventsList.pdf)

EDIT - I realize the 'Header' slider is the same on all of these pages. Both the image and color of the overlay are subject to change (although the emerald color looks much much better than any of the other SU colors available to us).

**Comment by reiners** [comment]
Hi Jack,
As far as the Contact form, whatever you think is best from a design/user experience perspective works for me. I like the list better than the calendar, since that gives you more room to add an image and more detail about the event.
Thanks!
Sara

Sara Reiner | Director of Marketing and Communications
School of New and Continuing Studies | Seattle University
901 12th Avenue, Seattle, WA 98122-1090
Office: (206) 296-2461
seattleu.edu/ncs<http://www.seattleu.edu/ncs>
Get Social with NCS
 [cid:6CC4EFAE-D00C-4170-8C5D-036F68F49201] <https://www.facebook.com/SeattleUNCS/> [cid:082190EA-60D0-46A8-AD8A-5C639D8DAAD3] <https://twitter.com/su_ncs> [cid:E499FA03-93B8-4F9B-B210-326A981B885E] <https://www.linkedin.com/company-beta/15208682>

From: Jack Fehrenbacher
Reply-To: suwebdev/ncs-michigan
Date: Thursday, February 9, 2017 at 11:40 AM
To: suwebdev/ncs-michigan
Cc: Sara Reiner, Mention
Subject: Re: [suwebdev/ncs-michigan] Present design proposal for child theme (#2)


As it stands right now the 'Contact' buttons in the theme default to a modal with a very simple form, but I'm sure we could either change the form in the child theme to reflect what's on the Request Information Page, or just have the 'Contact' button redirect to the Request Information page.

Here are two versions of the 'Upcoming Events' page. The Michigan Theme uses 'The Events Calendar' plugin, which can be displayed as a list, grid, or calendar. I went with the list and calendar formats here, but we could look at the grid version if you prefer.

UpcomingEventsCalendar.pdf<https://github.com/suwebdev/ncs-michigan/files/764793/UpcomingEventsCalendar.pdf>
UpcomingEventsList.pdf<https://github.com/suwebdev/ncs-michigan/files/764792/UpcomingEventsList.pdf>

**Comment by jfehrenbacher** [comment]
First swing at the WebDev Certificate program page. 3 versions for 3 header colors (emerald, green, blue). Thinking of trying to have a different header color for each program page. Let me know what you think when you get a chance!

[WebDev1Blue.pdf](https://github.com/suwebdev/ncs-michigan/files/765183/WebDev1Blue.pdf)
[WebDev1Emerald.pdf](https://github.com/suwebdev/ncs-michigan/files/765182/WebDev1Emerald.pdf)
[WebDev1Green.pdf](https://github.com/suwebdev/ncs-michigan/files/765184/WebDev1Green.pdf)

**Comment by shawnr** [comment]
Discussion in sprint shutdown:

* Really like whitespace and restful overall appearance.
* Much more modern and professional presentation. General appreciation all-around.

TODOs:
* Try replacing SU logo in top nav with a NCS logo of our own design. Use SU logo for link in global nav.
* Try a somewhat larger font for body text and see how it looks.

**Comment by shawnr** [comment]
Closing this issue, and moving revisions to #5
